### ISSUE TITLE
add persistent volume for '/opt/sonarqube/extensions'

### ISSUE THREAD

**Issue by rafaeltuelho** [OpenShiftDemos/sonarqube-openshift-docker#5]
Without a PV the installed plugins get lost on POD restart event...

**Comment by Curiouserw** [comment]
Yes 。Last weak，i didn't known this Issues。Today,I suffered the problem that always lost my SonarQube java plugin rulers .so i add mount persistence volumes at /opt/sonarqube/extensions directory in the template file。it work well。the thing i do just like you
